src/notification/notification.ts

```diff
@@ -38,7 +38,7 @@ export function sendNotification(link: Link, store: Store) {
 		sendDesktopNotification(link, store);
 	}
 
-	if (notifications.discord.webHookUrl) {
+	if (notifications.discord.webHookUrl.length > 0) {
```

Comment:
Right... Oops. Happened when I changed this to be lists. I'll start just putting up PRs now since we've grown quite a bit.

Thanks for catching this!